app/models/channel.rb

```diff
@@ -1,4 +1,3 @@
 class Channel < ApplicationRecord
-  belongs_to :inbox
```

Comment:
Don't believe this should be removed. 🤔

Reply:
inbox has chanel_id  attribute now, so assumed the data model changed now. 
I was a little confused about how the current data model is structured since I couldn't find a channels table in the schema. 

So ended up creating a new channel table for the seeds to pass.